src/euphorie/client/model.py

```diff
@@ -31,9 +31,8 @@
 from sqlalchemy import sql
 from sqlalchemy import types
 from sqlalchemy.event import listen
-from sqlalchemy.ext import declarative
-from sqlalchemy.orm import backref
-from sqlalchemy.orm import relationship
+from sqlalchemy.ext.declarative.extensions import instrument_declarative
+from sqlalchemy.orm.decl_base import _declarative_constructor
```

Comment:
Is it really necessary to import a private object? This might be unstable, right?

Reply:
Was it more stable before?
Do you require to change it?